common/compositor/gl/glprogram.cpp

```diff
@@ -130,24 +130,23 @@ static GLint LoadPreBuiltBinary(GLint gl_program,
                                 void *binary, long size) {
   GLint status;
 
-  std::string extensions((const char *)glGetString(GL_EXTENSIONS));
-  std::string mesa_prog_bin_fmts("GL_MESA_program_binary_formats");
-  std::string::size_type pos;
-
```

Comment:
Squash this hunk into the first commit, so you aren't adding code and then immediately removing it.